File: pertemuan 5/belajarOperator.py
a = input("Masukan nilai a =")

b = input("Masukan nilai b =")
print("A == B =" + str(a==b))
print("A != B =" + str(a!=b))
print("A > B =" + str(a > b))
print("A < B =" + str(a<b))
print("A >= B =" + str(a >= b))
print("A <= B =" + str(a <= b))

xAnd = int(a) < 5 and int(b) > 4
print("A < 5 and B > 4 =" + str(xAnd))
xOr = int(a) < 5 or int(b) > 4
print("A > 5 or B < 4 =" + str(xOr))
print("A < 5 not B > 4 =" + str(not(xAnd)))

File: pertemuan 5/belajarOperator.java
import java.util.Scanner;

public class belajarOperator {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Masukan nilai A =");
        double a = scanner.nextDouble();
        System.out.print("masukan nilai B =");
        double b = scanner.nextDouble();
        System.out.println("A == B =" + (a == b));
        System.out.println("A != B +" + (a != b));
        System.out.println("A > B =" + (a > b));
        System.out.println("A < B =" + ( a < b));
        System.out.println("A >= B =" + (a >= b));
        System.out.println("A <= B =" + (a <= b));
        boolean hasilAND = (a < 5 && b > 2);
        System.out.println("A < 5 && b > 2 =" + hasilAND);
        boolean hasilOR = (a < 5 || b > 2);
        System.out.println("A < 5 || B > 2 =" + hasilOR);
        System.out.println("!(A < 5 && B > 2) =" + !hasilAND);
    }
}
